### ISSUE TITLE
[Bug] Combining armors into relics gives out infinite dupes

### ISSUE THREAD

**Issue by memphismc** [PlayerEXDirectorsCut/RelicEx#8]
Fabric 1.20.1

https://github.com/PlayerEXDirectorsCut/RelicEx/assets/129392333/d030acc6-ef64-45f2-8829-2a0f93f077b9

**Comment by pokesmells** [comment]
Could you provide a mod list?

I cannot replicate it with just the base mods installed.

**Comment by memphismc** [comment]
> Could you provide a mod list?
> 
> I cannot replicate it with just the base mods installed.

yes, im sorry, its a modpack i came up with myself but it seems to be a bit problematic with the overenchanted mod as well, something is removing its functionality and i cant figure it out
cant figure out what makes the anvil act like this with relics either

https://drive.google.com/file/d/1xsrF2Y7_kfZNcVM6-1-KAS1VSV10Qm6e/view?usp=sharing

**Comment by memphismc** [comment]
update: so i found out that its either "all the trims" or "dynamic trims" that makes relics all the same texture, most likely dynamic trims i think

**Comment by memphismc** [comment]
> Could you provide a mod list?
> 
> I cannot replicate it with just the base mods installed.

hello again, so i've solved the issue with the overenchanted mod, it was goblin traders messing up the mod's functionality
unfortunately the duping issue with relicex is still happening for me and i cannot find out why for the life of me
please, if you can conduct some tests on your own let me know, i havent found anything on my own

**Comment by memphismc** [comment]
> Could you provide a mod list?
> 
> I cannot replicate it with just the base mods installed.

i have found the mod thats causing this, its stackables

**Comment by pokesmells** [comment]
Can you turn off each setting of stackables and see if it's a specific config or the mod itself?

**Comment by memphismc** [comment]
> Can you turn off each setting of stackables and see if it's a specific config or the mod itself?, I'm familiar with the mod

the mod has a config...?
no, i've uninstalled the mod for good now
also sorry but no i wont do that, i am tired of troubleshooting, the hate i feel for undiscolsed mixins inside random mods is bigger than ever right now
